comm/src/test/scala/coop/rchain/comm/transport/StreamHandlerSpec.scala

```diff
@@ -61,28 +61,6 @@ class StreamHandlerSpec extends FunSpec with Matchers with Inside {
       cache shouldBe empty
     }
 
-    it("should stop processing a stream if stream is missing part of header") {
```

Comment:
What is the intention to remove this test?

Reply:
This is relaetd to the fact that now sender in header cannot be `None`, so checking for an incomplete header does not make sense. A small discussion is provided [here](https://github.com/rchain/rchain/issues/3548#issuecomment-986209379) and in the response comment.